vrpn_Freespace.h.in

```diff
@@ -79,6 +137,10 @@ class VRPN_API vrpn_Freespace :
 	void handleUserFrame(const struct freespace_UserFrame&);
 	void handleBodyFrame(const struct freespace_BodyFrame&);
 	void handleLinkStatus(const struct freespace_LinkStatus&);
+#if FREESPACE_VERSION_GREATER_THAN(0,7,0)
+	void handleBodyUserFrame(const struct freespace_BodyUserFrame&);
```

Comment:
If you're using an older version of the library, you shouldn't call these functions (and wouldn't be ble tl anyway, since I'm only forward-declaring the structs). Will a simple comment to that effect suffice, do you think? Or is it better to have the compiler enforce it (as it does now)?